tests/unit_tests/test_tree.py

```diff
@@ -39,5 +43,105 @@ async def test_parsed_tree_empty_json():
         future.set_result(mocked_response)
         mocked_client.get.return_value = future
 
-        data = await parsed_tree(mocked_client, root_node_id)
+        data = await tree_from_vocabs(mocked_client, root_node_id)
         assert len(data) == 0
+
+
+def test_parse_tree():
+    node_id_biology = uuid.UUID("15fce411-54d9-467f-8f35-61ea374a298d")
+
+    query = tree_query(node_id_biology)
+    expected_query = {
+        "_source": [
+            "nodeRef.id",
+            "properties.cm:title",
+            "collections.path",
+            "parentRef.id",
+        ],
+        "from": 0,
+        "query": {
+            "bool": {
+                "filter": [{"term": {"path": node_id_biology}}],
+                "must": [
+                    {"match": {"permissions.Read": "GROUP_EVERYONE"}},
+                    {"match": {"properties.cm:edu_metadataset": "mds_oeh"}},
+                    {"match": {"nodeRef.storeRef.protocol": "workspace"}},
+                ],
+            }
+        },
+        "size": 500000,
+        "sort": ["fullpath"],
+    }
+
+    assert query.to_dict() == expected_query
+
+    if os.getenv("CI", False):
+        directory = "tests/unit_tests/resources"
+    else:
+        directory = "unit_tests/resources"
+
+    with open(f"{directory}/test_tree.json") as file:
+        expected_tree = json.loads("".join(file.readlines()))
```

Comment:
`json.load(file)` should do the trick as well?!?

Reply:
Yes, it does -_-